package.json

```diff
@@ -54,31 +54,31 @@
   ],
   "dependencies": {
     "async-foreach": "^0.1.3",
-    "chalk": "^1.1.1",
-    "cross-spawn": "^3.0.0",
-    "gaze": "^1.0.0",
-    "get-stdin": "^4.0.1",
+    "chalk": "^2.3.2",
+    "chokidar": "^2.0.3",
+    "cross-spawn": "^6.0.5",
```

Comment:
Do we still need this one if we don't have old Node?

Reply:
Good question. It works around a bunch of `child_process.spawn` issue, but they're maybe not relevant. Lets seen what CI thinks.

Reply:
Removing it made [Windows CI sad](https://ci.appveyor.com/project/sass/node-sass/build/2776/job/2d3xrs03rgvtgb75#L8877). There is a lot of code to work around [Windows in cross-spawn](https://github.com/moxystudio/node-cross-spawn/blob/master/lib/parse.js#L33-L65) which I'm happy to not have to deal with :)